tests/Type/DefinitionTest.php

```diff
@@ -702,19 +706,24 @@ public function testAllowsShorthandFieldDefinition() : void
 
         $schema = new Schema(['query' => $query]);
 
-        $valueField  = $schema->getType('SomeInterface')->getField('value');
-        $nestedField = $schema->getType('SomeInterface')->getField('nested');
+        /** @var InterfaceType $SomeInterface */
+        $SomeInterface = $schema->getType('SomeInterface');
 
+        $valueField  = $SomeInterface->getField('value');
         self::assertEquals(Type::string(), $valueField->getType());
+
+        $nestedField = $SomeInterface->getField('nested');
         self::assertEquals($interface, $nestedField->getType());
 
-        $withArg = $schema->getType('SomeInterface')->getField('withArg');
+        $withArg = $SomeInterface->getField('withArg');
         self::assertEquals(Type::string(), $withArg->getType());
 
         self::assertEquals('arg1', $withArg->args[0]->name);
         self::assertEquals(Type::int(), $withArg->args[0]->getType());
 
-        $testField = $schema->getType('Query')->getField('test');
+        /** @var ObjectType $Query */
+        $Query = $schema->getType('Query');
```

Comment:
small `Q`

Reply:
I think that in this case, having the variable name match the type name is fine. This is done in other parts of the lib as well.

Reply:
If it's in other parts of the lib, it should be fixed as well. 

camelCase for variable names is reasonably well-established in PHP world and others. There's not reason to break that. The only case I can think of is [naming tests using non-breaking spaces](https://mnapoli.fr/using-non-breakable-spaces-in-test-method-names/) / underscores for better in-tests-documentation readability.

Reply:
While i generally agree, in this case i like matching names better. Standards are rules are fine, but should not be applied dogmatically.

Reply:
The point is that if everyone who comes, makes PR with something he likes it will be a mess soon,I don't think there's any dogma at all. 

But makes me wonder why there's no CS rule for this. Will investigate.

Reply:
@simPod right. And right now, variables that contain types are named just like i did here throughout the test suite. Consistency is key.